src/main/java/microsoft/exchange/webservices/data/ExchangeServiceBase.java

```diff
@@ -98,19 +96,16 @@
   /**
    * The requested server version.
    */
-  private ExchangeVersion requestedServerVersion =
-      ExchangeVersion.Exchange2010_SP2;
+  private ExchangeVersion requestedServerVersion = ExchangeVersion.Exchange2010_SP2;
 
   /**
    * The server info.
    */
   private ExchangeServerInfo serverInfo;
 
-  private Map<String, String> httpHeaders = new HashMap<String, String>();
-
-  private Map<String, String> httpResponseHeaders = new HashMap<String, String>();
+  private Map<String, String> httpHeaders = new HashMap<>();
 
-  private TimeZone timeZone;
+  private Map<String, String> httpResponseHeaders = new HashMap<>();
```

Comment:
empty diamond operator causes the build to fail because it is a 1.7 feature. Maybe we should avoid this in the use of compatibility since it is just syntactic sugar.